ios/Podfile.lock

```diff
@@ -403,7 +403,7 @@ PODS:
     - React
   - react-native-create-thumbnail (1.6.4):
     - React-Core
-  - react-native-document-picker (8.2.2):
+  - react-native-document-picker (9.1.0):
```

Comment:
@yusufputra please change this react native document picker into 8.2.2 (without ^) by changing this in package.json

Reply:
updated mas @fajarism